ckanext/canada/tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+from ckanext.validation.model import create_tables, tables_exist
+
+def canada_tests_init_validation():
+ if not tables_exist():
+  create_tables()
```

Comment:
1-space indents?

Reply:
This would be better in a FunctionalTestBase subclass or a fixture for our tests

Reply:
Huh, my editor config is set to `*.py` but must not work on the `__init__.py` files, interesting.

Will make into a fixture if 2.8 + py2 allows it. Otherwise will make it into a subclass.

Reply:
@wardi Fixture not supported for this version, so just sub classed it as `CanadaTestBase`